test/wrap-loader.js

```diff
@@ -0,0 +1,10 @@
+import loader from '../src/cjs';
```

Comment:
👍  Put `wrap-loader.js` && `webpack.js` into a e.g `utils` folder and rename `wrap-loader => loader.js` please

```
|– test
| |– __snapshots__
| |– index.test.js
| |– ${option.name}.test.js
| |– ...
| |– utils
| | |– webpack.js (Compiler/Runner)
| | |– stats.js (Generic Stats/Compilation Normalizer [ TODO ])
| | |– loader.js (Generic Loader Normalizer)
``` 

Sry to bother you with this all the time, it's drafting out the base test system on the job :D